Tests/resources/directory/twig/subdirectory/index.html.twig

```diff
@@ -0,0 +1,4 @@
+<p>Any not translated text</p>
+And not in a HTML tag
+
+{{ 'Translate me please'|trans({}, 'Test.Translation.FirstSubDir') }}
```

Comment:
CamelCase within domain components is not allowed. It will work here but would set a bad example
```suggestion
{{ 'Translate me please'|trans({}, 'Test.Translation.Firstsubdir') }}
```